example/package.json

```diff
@@ -5,16 +5,28 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "prop-types": "^15.6.2",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
-    "react-scripts": "^1.1.4",
-    "react-stories-api": "link:.."
+    "prop-types": "^15.7.2",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-scripts": "^3.2.0",
+    "react-stories-api": "file:.."
```

Comment:
The change from "link" -> "file" may be a Mac-only fix, so I'm thinking we can merge this for now, but keep an eye out for it in production and on non-mac computers